CONTRIBUTING.md

```diff
@@ -7,75 +7,66 @@ The following are a set of guidelines for contributing to the project.
 #### Contents
 
 * [Code of Conduct](#code-of-conduct)
-* [Ways of Contributing](#ways-of-contributing)
-  * [Joining Feature Discussions](#joining-feature-discussions-) 💭
-  * [Joining Developer Discussions](#joining-developer-discussions-) 💬
-  * [Reporting Bugs](#reporting-bugs-) 🐛
-  * [Requesting Features](#requesting-features-) 🚀
+* [Ways to Contribute](#ways-to-contribute)
+  * [Questions or Enhancement Requests](#questions-or-enhancement-requests-) 💭
+  * [Reporting and Fixing Bugs](#reporting-and-fixing-bugs-) 🐛
+  * [Submitting Features](#submitting-features-) 🚀
   * [Contributing Code](#contributing-code-) 👾
   * [Reviewing Code](#reviewing-code-) 👀
   * [Documenting](#documenting-) 📜
+  * [Joining Feature Discussions](#joining-feature-discussions-) 💭
```

Comment:
@glasert - are these two sections still relevant or could they be merged into the new "Questions or Enhancement Requests" section?

Reply:
I also think we should move the "Questions or Enhancement Requests" section to the end rather than having it as the first way to contribute